app.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+const fetch = require('node-fetch')
+const colors = require('colors')
```

Comment:
Esta cool el uso de const para variables que no mutan su valor.
```suggestion
const fs = require('fs')
const fetch = require('node-fetch')
const colors = require('colors')
```